src/PHPUnit/Comparator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace MyCLabs\Enum\PHPUnit;
+
+use MyCLabs\Enum\Enum;
+use SebastianBergmann\Comparator\ComparisonFailure;
```

Comment:
To be consistency and avoid namespace collision, consider following use namespace syntax:

```php
use SebastianBergmann\Comparator\Comparator as SebastianBergmannComparator
```

Reply:
No need for that, using the FQN is already clear enough in the `extends`.